docs/moonlight/drivers.md

```
@@ -93,17 +93,18 @@ Sends Lights in Art-Net compatible packages to an Art-Net controller specified b
 
 #### Controls
 
-* **Controller IP**: The last segment of the IP address within your local network, of the the hardware Art-Net controller.
+* **Controller IPs**: The last segment of the IP address within your local network, of the hardware Art-Net controller. Add more IPs if you send to more than one controller, comma separated.
 * **Port**: The network port added to the IP address, 6454 is the default for Art-Net.
 * **FPS Limiter**: set the max frames per second Art-Net packages are send out (also all the other nodes will run at this speed).
     * Art-Net specs recommend about 44 FPS but higher framerates will work mostly (up to until ~130FPS tested)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)